app/helpers/organization_helper.rb

```diff
@@ -2,6 +2,6 @@
 module OrganizationHelper
   def display_logo_or_name(organization = nil)
     organization ||= current_organization
-    organization.logo.attached? ? image_tag(organization.logo, alt: "#{organization.name} logo", class: "organization-logo", style: "max-height:188px") : organization.name
+    organization.logo.attached? ? image_tag(organization.logo, alt: "#{organization.name} logo", class: "organization-logo", style: "max-height:40px") : ""
```

Comment:
@kre8sions Since there's a lot of style edits in this anyways, would it make more sense to put this into the SCSS instead of doing an override here? Or was the local styling done intentionally as an override? (@seanmarcia  do you remember? I think you added this)

Reply:
Same. I don't remember this either :(

Reply:
I'm getting old.d